cli/examples/accessibility-onClosedCaptionsSettings2.event.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Please change the name of this file to match the method name... e.g., add "Changed" to the file name.

Reply:
done